src/Sylius/Bundle/AdminBundle/Resources/config/grids/promotion.yml

```diff
@@ -51,6 +51,12 @@ sylius_grid:
                     label: sylius.ui.coupon
                     options:
                         fields: [coupons.code]
+                archival:
```

Comment:
Why does grid has archive field since record/item won't appear until correct filter will be selected? Do you have a screenshot of what it does?

Reply:
It is not a field. It's a filter.

Reply:
Oops :sweat_smile: